.gitignore

```diff
@@ -8,6 +8,7 @@ node_modules
 
 # production
 /build
+public/build-info.json
```

Comment:
Is there a way to get this file into the static server data without putting it in a source tree location? It'd be nice if we could use `build` or some other generated source location instead.

Reply:
OK, thanks for the explanation. Given these options leaving it in gitignore seems fine to me.